src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -10,6 +10,10 @@
     },
     "frameworks": {
         "net451": {
+            "imports": ["net46"],
```

Comment:
I get the following restore error if I don't have that import:

Errors in C:\projects\cli_2\src\Microsoft.DotNet.Cli.Utils\project.json
    System.AppContext 4.1.0-rc2-23727 is not compatible with .NETFramework,Version=v4.5.1.
    Some packages are not compatible with .NETFramework,Version=v4.5.1.
    System.AppContext 4.1.0-rc2-23727 is not compatible with .NETFramework,Version=v4.5.1 (win7-x64).
    Some packages are not compatible with .NETFramework,Version=v4.5.1 (win7-x64).
    System.AppContext 4.1.0-rc2-23727 is not compatible with .NETFramework,Version=v4.5.1 (win7-x86).
    Some packages are not compatible with .NETFramework,Version=v4.5.1 (win7-x86).

---
In reply to: [56082599](https://github.com/dotnet/cli/pull/1847#discussion_r56082599) [](ancestors = 56082599)